defaults/main.yml

```diff
@@ -527,6 +527,12 @@ gitlab_runner__cache_secret_key: ''
 gitlab_runner__cache_bucket_name: ''
 
                                                                    # ]]]
+# .. envvar:: gitlab_runner__cache_bucket_location [[[
+#
+# Cache bucket location
+gitlab_runner__cache_bucket_location: 'us-east-1'
```

Comment:
A lot of AWS things default to us-east-1. This is also the default region S3 buckets are created (US Standard). Thoughts on leaving the default or just leave it blank?